2021-11-16 22:10:14,521 INFO    MainThread:1951 [wandb_setup.py:_flush():69] setting env: {'project': 'exercise_14', 'run_group': 'dev'}
2021-11-16 22:10:14,521 INFO    MainThread:1951 [wandb_setup.py:_flush():69] setting login settings: {}
2021-11-16 22:10:14,521 INFO    MainThread:1951 [wandb_init.py:_log_setup():334] Logging user logs to /Users/sukanto/WD/genre_classification/preprocess/wandb/run-20211116_221014-20cuejum/logs/debug.log
2021-11-16 22:10:14,521 INFO    MainThread:1951 [wandb_init.py:_log_setup():335] Logging internal logs to /Users/sukanto/WD/genre_classification/preprocess/wandb/run-20211116_221014-20cuejum/logs/debug-internal.log
2021-11-16 22:10:14,521 INFO    MainThread:1951 [wandb_init.py:init():367] calling init triggers
2021-11-16 22:10:14,521 INFO    MainThread:1951 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-11-16 22:10:14,522 INFO    MainThread:1951 [wandb_init.py:init():416] starting backend
2021-11-16 22:10:14,522 INFO    MainThread:1951 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-11-16 22:10:14,538 INFO    MainThread:1951 [backend.py:ensure_launched():123] starting backend process...
2021-11-16 22:10:14,543 INFO    MainThread:1951 [backend.py:ensure_launched():127] started backend process with pid: 1959
2021-11-16 22:10:14,545 INFO    MainThread:1951 [wandb_init.py:init():421] backend started and connected
2021-11-16 22:10:14,547 INFO    MainThread:1951 [wandb_init.py:init():461] updated telemetry
2021-11-16 22:10:14,547 INFO    MainThread:1951 [wandb_init.py:init():480] communicating current version
2021-11-16 22:10:15,182 INFO    MainThread:1951 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.12.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-16 22:10:15,182 INFO    MainThread:1951 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-11-16 22:10:16,098 INFO    MainThread:1951 [wandb_init.py:init():518] starting run threads in backend
2021-11-16 22:10:16,136 INFO    MainThread:1951 [wandb_run.py:_console_start():1476] atexit reg
2021-11-16 22:10:16,136 INFO    MainThread:1951 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2021-11-16 22:10:16,136 INFO    MainThread:1951 [wandb_run.py:_redirect():1344] Redirecting console.
2021-11-16 22:10:16,136 INFO    MainThread:1951 [redirect.py:install():213] install start
2021-11-16 22:10:16,137 INFO    MainThread:1951 [redirect.py:install():228] install stop
2021-11-16 22:10:16,137 INFO    MainThread:1951 [redirect.py:install():213] install start
2021-11-16 22:10:16,137 INFO    MainThread:1951 [redirect.py:install():228] install stop
2021-11-16 22:10:16,137 INFO    MainThread:1951 [wandb_run.py:_redirect():1390] Redirects installed.
2021-11-16 22:10:16,137 INFO    MainThread:1951 [wandb_init.py:init():542] run started, returning control to user process
2021-11-16 22:10:25,680 INFO    MainThread:1951 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2021-11-16 22:10:25,680 INFO    MainThread:1951 [wandb_run.py:_restore():1418] restore
2021-11-16 22:10:25,680 INFO    MainThread:1951 [redirect.py:uninstall():232] uninstall start
2021-11-16 22:10:25,680 INFO    MainThread:1951 [redirect.py:_stop():287] _stop: stdout
2021-11-16 22:10:25,681 INFO    MainThread:1951 [redirect.py:_stop():293] _stop closed: stdout
2021-11-16 22:10:25,681 INFO    stdout    :1951 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-11-16 22:10:25,681 INFO    stdout    :1951 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-11-16 22:10:25,681 INFO    MainThread:1951 [redirect.py:_stop():299] _stop joined: stdout
2021-11-16 22:10:25,681 INFO    MainThread:1951 [redirect.py:_stop():301] _stop rd closed: stdout
2021-11-16 22:10:25,681 INFO    MainThread:1951 [redirect.py:uninstall():236] uninstall done
2021-11-16 22:10:25,681 INFO    MainThread:1951 [redirect.py:uninstall():232] uninstall start
2021-11-16 22:10:25,681 INFO    MainThread:1951 [redirect.py:_stop():287] _stop: stderr
2021-11-16 22:10:25,681 INFO    MainThread:1951 [redirect.py:_stop():293] _stop closed: stderr
2021-11-16 22:10:25,681 INFO    stderr    :1951 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-11-16 22:10:25,681 INFO    stderr    :1951 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-11-16 22:10:25,681 INFO    MainThread:1951 [redirect.py:_stop():299] _stop joined: stderr
2021-11-16 22:10:25,681 INFO    MainThread:1951 [redirect.py:_stop():301] _stop rd closed: stderr
2021-11-16 22:10:25,682 INFO    MainThread:1951 [redirect.py:uninstall():236] uninstall done
2021-11-16 22:10:36,838 INFO    MainThread:1951 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 6735894
  total_bytes: 6735894
}

2021-11-16 22:10:38,845 INFO    MainThread:1951 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 6736774
  total_bytes: 6736774
}

2021-11-16 22:10:39,930 INFO    MainThread:1951 [wandb_run.py:_show_files():1790] logging synced files
